lib/src/feature/onboarding/splash/view/splash_screen.dart

```diff
@@ -54,16 +55,18 @@ class _SplashScreenState extends ConsumerState<SplashScreen> {
   void _navigateToLandingScreen() {
     Navigator.pushReplacement(
       context,
+      // ignore: always_specify_types
```

Comment:
Do a search in vscode (or whatever editor you're using) and remove all instances of `// ignore: always_specify_types `then you can easily see what files you have missed.